.github/workflows/release.yml

```
@@ -84,11 +84,27 @@ jobs:
         uses: google-github-actions/setup-gcloud@v2
         with:
           version: "469.0.0"
-      - name: Upload zip
+      - name: Upload zip to GCP
         uses: google-github-actions/upload-cloud-storage@v2
         with:
           path: ${{ env.OUTNAME }}.zip
           destination: ${{ secrets.GCP_BUCKET }}/${{ github.ref_name }}/
+      - name: Copy build
+        shell: bash
+        run: |
+          mkdir build-zip
+          cp ${{ env.OUTNAME }}.zip build-zip
+      - name: Upload zip to R2
+        uses: jakejarvis/s3-sync-action@master
+        with:
+          args: --acl public-read --follow-symlinks
+        env:
+            AWS_S3_BUCKET: ${{ secrets.CLOUDFLARE_GO_SM_BUILDS_BUCKET }}
+            AWS_ACCESS_KEY_ID: ${{ secrets.CLOUDFLARE_GO_SM_BUILDS_ACCESS_KEY_ID }}
+            AWS_SECRET_ACCESS_KEY: ${{ secrets.CLOUDFLARE_GO_SM_BUILDS_SECRET_ACCESS_KEY }}
+            SOURCE_DIR: build-zip
+            DEST_DIR: '${{ github.ref_name }}'
+            AWS_S3_ENDPOINT: https://${{ secrets.CLOUDFLARE_ACCOUNT_ID }}.r2.cloudflarestorage.com
```

I don't think we need the action at all. This does the same (aws CLI already installed on GH runners):

```suggestion
      - name: Upload zip to R2
        run: >
          aws s3 sync build-zip
          s3://${{ secrets.CLOUDFLARE_GO_SM_BUILDS_BUCKET }}/${{ github.ref_name }}
          --endpoint-url https://${{ secrets.CLOUDFLARE_ACCOUNT_ID }}.r2.cloudflarestorage.com
          --acl public-read --follow-symlinks --delete

        env:
            AWS_ACCESS_KEY_ID: ${{ secrets.CLOUDFLARE_GO_SM_BUILDS_ACCESS_KEY_ID }}
            AWS_SECRET_ACCESS_KEY: ${{ secrets.CLOUDFLARE_GO_SM_BUILDS_SECRET_ACCESS_KEY }}
            AWS_REGION: us-east-1
```